### DISCUSSION TITLE
이력서 지원시 무지성으로 일단 넣고! 보는게 맞을까요?

### DISCUSSION THREAD

**Discussion by gksgpwls0306** [biancasuji/biancasuji_#5] [Q&A] [answered]
이력서 지원시 무지성으로 일단 넣고! 보는게 맞을까요? 이것저것 보고 넣는게 맞을까요?

**Comment by biancasuji** [comment, marked as answer]
움.. 회사를 너무 막 넣는것보다는 괜찮은 회사 준비해서 쫘악 넣고 그 다음에 2순위 회사들 쫘라라락 넣는걸로 해보아요!!!
